Add code templates to issue template

Comandeer opened the pull request:
## What is the purpose of this pull request?

Task

## Does your PR contain necessary tests?

All patches which change the editor code must include tests. You can always read more
on [PR testing](https://ckeditor.com/docs/ckeditor4/latest/guide/dev_contributing_code.html#tests),
[how to set the testing environment](https://ckeditor.com/docs/ckeditor4/latest/guide/dev_tests.html) and
[how to create tests](https://ckeditor.com/docs/ckeditor4/latest/guide/dev_tests.html#creating-your-own-test)
in the official CKEditor documentation.

### This PR contains

- [ ] Unit tests
- [ ] Manual tests

## What is the proposed changelog entry for this pull request?

No changelog entry needed.

## What changes did you make?

I've added a short info about adding reproductions with a link to CodePen sample (based on https://github.com/ckeditor/ckeditor4/issues/3790#issuecomment-575141543). However I'm not sure if the sample is not too complex 🤔 I'm also thinking about adding another services, like JSFiddle and CodeSandbox (for more complex bugs).

## Which issues your PR resolves?

Closes #3790.

jacekbogdanski commented:
I would also add https://jsbin.com/ as it supports IE and it's pretty lightweight.

Comandeer commented:
I've added other samples: JSFiddle, JSBin and StackBlitz (instead of CodeSandbox, which didn't want to work correctly). I've also added simple instructions in JS code. I don't know how to change editor's content however 🤔 Maybe just leave it as is?
